apiserver/applyconfigurations/apiserver/v1/apirequestcount.go

```diff
@@ -30,41 +27,6 @@ func APIRequestCount(name string) *APIRequestCountApplyConfiguration {
 	return b
 }
 
-// ExtractAPIRequestCount extracts the applied configuration owned by fieldManager from
-// aPIRequestCount. If no managedFields are found in aPIRequestCount for fieldManager, a
-// APIRequestCountApplyConfiguration is returned with only the Name, Namespace (if applicable),
-// APIVersion and Kind populated. It is possible that no managed fields were found for because other
-// field managers have taken ownership of all the fields previously owned by fieldManager, or because
-// the fieldManager never owned fields any fields.
-// aPIRequestCount must be a unmodified APIRequestCount API object that was retrieved from the Kubernetes API.
-// ExtractAPIRequestCount provides a way to perform a extract/modify-in-place/apply workflow.
-// Note that an extracted apply configuration will contain fewer fields than what the fieldManager previously
-// applied if another fieldManager has updated or force applied any of the previously applied fields.
-// Experimental!
-func ExtractAPIRequestCount(aPIRequestCount *apiapiserverv1.APIRequestCount, fieldManager string) (*APIRequestCountApplyConfiguration, error) {
-	return extractAPIRequestCount(aPIRequestCount, fieldManager, "")
-}
-
-// ExtractAPIRequestCountStatus is the same as ExtractAPIRequestCount except
-// that it extracts the status subresource applied configuration.
-// Experimental!
-func ExtractAPIRequestCountStatus(aPIRequestCount *apiapiserverv1.APIRequestCount, fieldManager string) (*APIRequestCountApplyConfiguration, error) {
-	return extractAPIRequestCount(aPIRequestCount, fieldManager, "status")
-}
-
-func extractAPIRequestCount(aPIRequestCount *apiapiserverv1.APIRequestCount, fieldManager string, subresource string) (*APIRequestCountApplyConfiguration, error) {
-	b := &APIRequestCountApplyConfiguration{}
-	err := managedfields.ExtractInto(aPIRequestCount, internal.Parser().Type("com.github.openshift.api.apiserver.v1.APIRequestCount"), fieldManager, b, subresource)
-	if err != nil {
-		return nil, err
-	}
-	b.WithName(aPIRequestCount.Name)
-
-	b.WithKind("APIRequestCount")
-	b.WithAPIVersion("apiserver.openshift.io/v1")
-	return b, nil
-}
-
```

Comment:
This removal implies to me that the openapi configuration isn't being passed correctly to the applyconfiguration generator

Reply:
Fair, looking into it...